grafana/grafana.example.ini

```diff
@@ -0,0 +1,14 @@
+#################################### SMTP / Emailing #####################
+[smtp]
+enabled = true
```

Comment:
examples would be nice for some of the settings,

Reply:
you're right, also if (in the future) some other part of grafana.ini (can be very long) defines a password the substitution would be messed up. imrpoved the substitution in [b74b982](https://github.com/MaRDI4NFDI/portal-compose/pull/162/commits/b74b98235b1ccd2033dbbf0b1a8fc96df1f187a6)